cmd/crc/cmd/podman_env.go

```diff
@@ -45,6 +45,5 @@ func RunPodmanEnv(args []string) error {
 }
 
 func init() {
-	rootCmd.AddCommand(podmanEnvCmd)
```

Comment:
?

Reply:
According to telemetry, lot of people try this command. This is disappointing. We should remove it if it doesn't work.

Reply:
or fix it ?

Reply:
I don't think we should fix it.
* It will be hard to fix it before next release
* The version of podman included with the bundle is too old to be interesting for the end user.

Reply:
I dropped this commit.